<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Lab3" ignored="0" name="test" tests="1" failures="0" timestamp="08 abr 2019 10:18:01 GMT" time="0.059" errors="0">
  <testcase name="@BeforeMethod init" time="0.006" classname="testingauto.Tests">
    <failure type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
at events.EventHandling.<init>(EventHandling.java:17)
at testingauto.Tests.init(Tests.java:97)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
... Removed 13 stack frames]]>
    </failure>
  </testcase> <!-- @BeforeMethod init -->
  <testcase name="@BeforeMethod runTest" time="0.0" classname="testingauto.Tests">
    <skipped/>
  </testcase> <!-- @BeforeMethod runTest -->
  <testcase name="@AfterClass tearDown" time="0.0" classname="testingauto.Tests">
    <skipped/>
  </testcase> <!-- @AfterClass tearDown -->
  <testcase name="reserveDefaultFlightTest" time="0.0" classname="testingauto.Tests">
    <skipped/>
  </testcase> <!-- reserveDefaultFlightTest -->
</testsuite> <!-- test -->
